### PR TITLE
Document `Is.WhiteSpace` and `IgnoreWhiteSpace`

### PR DESCRIPTION
**Pull request by manfred-brands**
Documentation update belonging to: https://github.com/nunit/nunit/pull/4664

### CODE REVIEW

**Comment by manfred-brands** [comment]
@SeanKilleen Do we put any version information in the documentation?

I saw `...UsingPropertiesComparer()  // From version 4.1`

Should I changed my updates to be `...IgnoreWhiteSpace // From version 4.2`?

**Comment by manfred-brands** [comment]
The PR is merged.

**Comment by OsirisTerje** [comment]
@manfred-brands 

> Should I changed my updates to be ...IgnoreWhiteSpace // From version 4.2?

Yes, please

File: docs/articles/nunit/writing-tests/constraints/CollectionEquivalentConstraint.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ Is.EquivalentTo(IEnumerable other)
 ## Modifiers
 
 ```csharp
+...IgnoreCase
+...IgnoreWhiteSpace
```
Comment (OsirisTerje) [inline docs/articles/nunit/writing-tests/constraints/CollectionEquivalentConstraint.md:22]:
```suggestion
...IgnoreWhiteSpace  // From version 4.2
```

Comment (manfred-brands) [reply docs/articles/nunit/writing-tests/constraints/CollectionEquivalentConstraint.md]: Done

**Pull request merged**